pkgs/development/tools/misc/autoconf/2.13.nix

```diff
@@ -9,6 +9,7 @@ stdenv.mkDerivation rec {
     sha256 = "07krzl4czczdsgzrrw9fiqx35xcf32naf751khg821g5pqv12qgh";
   };
 
+  nativeBuildInputs = [ m4 ];
   buildInputs = [ m4 perl ];
```

Comment:
Did you check if we still need this?

Reply:
Moved both `m4` and `perl` to `nativeBuildInputs`. `diffoscope` says there is no output difference.